skimage/morphology/_skeletonize_cy.pyx

```diff
@@ -3,7 +3,109 @@
 #cython: nonecheck=False
 #cython: wraparound=False
 
-'''
+import numpy as np
+cimport numpy as cnp
+
+def _fast_skeletonize(image):
+    """Optimized parts of the Zhang-Suen skeletonization.
```

Comment:
blank lines between paragraphs.

Also, can you add a reference to the original paper? I didn't know that's what this algorithm was called. References are done as follows:

```
Zhang-Suen skeletonization [1]_.

References
----------
.. [1] K Zhang and F Suen. Skeletonizing binary images. (1950) Journal of Awesome 1:1-10.
```